src/main/java/fi/jubic/quanta/models/view/ExternalClientView.java

```diff
@@ -0,0 +1,50 @@
+package fi.jubic.quanta.models.view;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import fi.jubic.easyvalue.EasyValue;
+import fi.jubic.quanta.models.ExternalClient;
+import fi.jubic.quanta.models.Task;
+
+import javax.annotation.Nullable;
+
+@EasyValue
+@JsonDeserialize(builder = ExternalClientView.Builder.class)
+public abstract class ExternalClientView {
+
+    public abstract Long getId();
+
+    public abstract String getName();
+
+    public abstract String getToken();
+
+    @Nullable
```

Comment:
Use an empty string instead of nullable.

Reply:
Removed @Nullable